Extension/src/LanguageServer/client.ts

```diff
@@ -620,6 +627,16 @@ class DefaultClient implements Client {
         this.model.tagParserStatus.Value = notificationBody.status;
     }
 
+    private updateInactiveRegions(params: InactiveRegionParams): void {
+        let renderOptions: vscode.DecorationRenderOptions = {
+            color: "rgba(255,0,255,0.5)"
```

Comment:
>rgba(255,0,255,0.5) [](start = 20, length = 19)

it looks like we still have the test color in here.  We need to remember to make a light-themed and dark-themed gray color here,

Reply:
Most definitely. I will add different coloration based on light/dark, and I will keep note of it as a discussion point when we talk as a group, and when find out more about the possibility of opacity.